include/kos/init.h

```diff
@@ -139,29 +139,33 @@ extern const void * __kos_romdisk;
     @{
 */
 
-/** Init flags to include all virtual filesystems (default). */
-#define INIT_FS_ALL     (INIT_FS_ROMDISK | INIT_FS_NULL | INIT_FS_PTY | \
-                         INIT_FS_RAMDISK | INIT_FS_RND  | INIT_FS_DEV)
-
 /** Default init flags (IRQs on, preemption enabled, romdisks). */
 #define INIT_DEFAULT    (INIT_IRQ     | INIT_THD_PREEMPT | INIT_FS_ALL | \
                          INIT_LIBRARY | INIT_DEFAULT_ARCH)
 
+/** Init flags to include all virtual filesystems within `/dev` */
+#define INIT_FS_DEV     (INIT_FS_NULL | INIT_FS_RND)
+
+/** Init flags to include all virtual filesystems (default). */
+#define INIT_FS_ALL     (INIT_FS_ROMDISK | INIT_FS_RAMDISK | \
+                         INIT_FS_PTY     | INIT_FS_DEV)
+
 #define INIT_NONE        0x00000000  /**< Don't init optional things */
+#define INIT_THD_PREEMPT 0x00000000  /**< \deprecated Already default mode */
 #define INIT_IRQ         0x00000001  /**< Enable IRQs at startup */
-#define INIT_THD_PREEMPT 0x00000002  /**< \deprecated Already default mode */
-#define INIT_NET         0x00000004  /**< Enable built-in networking */
-#define INIT_MALLOCSTATS 0x00000008  /**< Enable malloc statistics */
-#define INIT_QUIET       0x00000010  /**< Disable dbgio */
-#define INIT_EXPORT      0x00000020  /**< Export kernel symbols */
-#define INIT_LIBRARY     0x00000040  /**< Enable support for dynamic libraries. */
-#define INIT_FS_ROMDISK  0x00000080  /**< Enable support for romdisks */
-#define INIT_FS_NULL     0x00000100  /**< Enable support for /dev/null VFS */
-#define INIT_FS_PTY      0x00000200  /**< Enable support for PTY VFS */
-#define INIT_FS_RAMDISK  0x00000400  /**< Enable support for ramdisk VFS */
-#define INIT_FS_RND      0x00000800  /**< Enable support for /dev/urandom VFS */
-#define INIT_FS_DEV      0x00001000  /**< Enable support for /dev/ VFS */
-#define INIT_NO_SHUTDOWN 0x00002000  /**< Disable hardware shutdown */
+#define INIT_NET         0x00000002  /**< Enable built-in networking */
+#define INIT_MALLOCSTATS 0x00000004  /**< Enable malloc statistics */
+#define INIT_QUIET       0x00000008  /**< Disable dbgio */
+#define INIT_EXPORT      0x00000010  /**< Export kernel symbols */
+#define INIT_LIBRARY     0x00000020  /**< Enable support for dynamic libraries. */
```

Comment:
`INIT_EXPORT` and `INIT_LIBRARY` is the same subsystem. Why does it need separate flags?

Reply:
Ehhhhh... not quite. One of them is for exporting KOS's symbols for external libraries to make KOS calls, the other is for just loading external libraries themselves, which could be used without needing to link back to the main binary to make such KOS calls...

But now that you are mentioning it, I suppose that would be pretty silly, as... what the heck are you going to do in a dynamic library with access to essentially nothing? Okay, point taken. Will combine the two when I get home, but I would prefer `INIT_LIBRARY` then to handle both, as `INIT_EXPORT` doesn't seem as descriptive or to include the dynamic library loader itself, based on the name.

Reply:
Better not break the API for no reason though.

Reply:
Alrighty, that's legit with me. Good call.

Reply:
> Ehhhhh... not quite. One of them is for exporting KOS's symbols for external libraries to make KOS calls, the other is for just loading external libraries themselves, which could be used without needing to link back to the main binary to make such KOS calls...
> 
> But now that you are mentioning it, I suppose that would be pretty silly, as... what the heck are you going to do in a dynamic library with access to essentially nothing? Okay, point taken. Will combine the two when I get home, but I would prefer `INIT_LIBRARY` then to handle both, as `INIT_EXPORT` doesn't seem as descriptive or to include the dynamic library loader itself, based on the name.

So you won't load the library without exports. I mean, it can't do much without the KOS, stdlib and other APIs that need to be exported.

Reply:
Alrighty, values have been readjusted to be the same for `INIT_EXPORT` and `INIT_LIBRARY`.